core/src/ldml/ldml_processor.cpp

```diff
@@ -259,7 +260,8 @@ void ldml_event_state::emit_backspace() {
     dumped in somewhere unknown, so we will have to depend on the app to
     be sensible about backspacing because we know nothing.
   */
-  state->actions().push_backspace(KM_CORE_BT_UNKNOWN);
+  // TODO-LDML: What here?
+  // state->actions().push_backspace(KM_CORE_BT_UNKNOWN);
```

Comment:
This should be `emit_keystroke = 1` I think

Reply:
1-ish